website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -36,7 +36,7 @@ Sign up on [customer.appsmith.com](https://customer.appsmith.com/) and generate
 5. To retrieve the salt and password from the pod, run the following command and copy the values to `values.yaml`.
 
    ```bash
-   kubectl exec <pod_name> -- grep /appsmith-stacks/configuration/docker.env APPSMITH_ENCRYPTION_
+   kubectl exec <pod_name> -- grep APPSMITH_ENCRYPTION_ /appsmith-stacks/configuration/docker.env
```

Comment:
@pratapaprasanna - Please review.

Reply:
@pratapaprasanna - as we are in the process of rehauling the doc, we will point to appropriate sections in the [Backup and Restore](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl) doc. Please confirm whether the below command should work, or do we need to update it as per this PR?

```
kubectl exec ANY_APPSMITH_POD_NAME cat /appsmith-stacks/configuration/docker.env | grep ENCRYPTION
```

Reply:
@jnikhila This command should work as well, but you should add a double dash between pod name and command:
```
kubectl exec ANY_APPSMITH_POD_NAME -- cat /appsmith-stacks/configuration/docker.env | grep ENCRYPTION
```
Usage without double dash is deprecated.

Reply:
Thanks @alexalex89.

Reply:
Thanks, @pratapaprasanna - I'll make these command changes to backup and restore doc in the same PR, as the upgrade to the business edition K8s page will point to these instructions. 
Thanks, @alexalex89 for raising this.